cmds/core/tail/tail.go

```diff
@@ -267,26 +267,44 @@ func run(reader *os.File, writer io.Writer, follow bool, numLines int, followDur
 		inFile *os.File
 		err    error
 	)
-	switch len(args) {
-	case 0:
-		inFile = reader
-	case 1:
-		inFile, err = os.Open(args[0])
-		if err != nil {
-			return err
-		}
-	default:
-		// TODO support multiple files
-		return fmt.Errorf("tail: can only read one file at a time")
-	}
 
 	// TODO: add support for parsing + (from beggining of the file)
 	// negative sign is the same as none
 	if numLines < 0 {
 		numLines = -1 * numLines
 	}
 	config := tailConfig{follow: follow, numLines: uint(numLines), followDuration: followDuration}
-	return tail(inFile, writer, config)
+
+	switch len(args) {
+	case 0:
+		return tail(reader, writer, config)
+	default:
+		if follow {
```

Comment:
there would be a lot of value for follow for multiple files, however. Maybe in future, if follow is set, the various tails could be run as goroutines?

Reply:
yeah, I plan do that latter